assets/manifest.json

```diff
@@ -25,22 +25,22 @@
   "content_scripts": [
     {
       "matches": [
-        "https://github.com/*",
         "https://gist.github.com/*"
       ],
       "js": [
-        "app.js"
-      ],
-      "css": [
-        "style.css"
+        "empty.js"
```

Comment:
At least one script must be defined otherwise the extension will be invalid. Therefore I added an empty file. Will revert that to later.